.github/workflows/ci-lint.yml

```
@@ -0,0 +1,45 @@
+  lint:
+    name: lint
+    runs-on: unbutu-22.04
+    steps:
+      - name: 🔄 Checkout
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
+
+      - name: 🦀 Install rust stable
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: rustfmt, clippy
+          targets: ${{ matrix.os == 'macos-latest' && 'aarch64-apple-darwin,x86_64-apple-darwin,wasm32-unknown-unknown' || 'wasm32-unknown-unknown' }}
```

actionlint (1.7.8)

property "os" is not defined in object type {}

(expression)